packages/config/src/VitessceAutoConfig.test.js

```diff
@@ -447,12 +447,98 @@ describe('src/VitessceAutoConfig.js', () => {
   });
 
 
-  it('raises an Error when .zmetadata file not present in folder', async () => {
-    const urls = ['http://localhost:51204/@fixtures/zarr/partials/invalid.adata.zarr'];
+  it('Does the parsing when .zmetadata file not present in folder', async () => {
+    const urls = ['http://localhost:51204/@fixtures/zarr/anndata-0.8/anndata-csr.adata.zarr'];
```

Comment:
I needed all the renames because of this unit test. If I don't rename, the unit test fails because the file extension is not `adata.zarr`